content/blog/gsoc-25-netjsongraph.rst

```diff
@@ -0,0 +1,225 @@
+GSoC 2025: Improving the netjsongraph.js Visualization Library
+==============================================================
+
+:date: 2025-09-22
+:author: cestercian
+:tags: gsoc, netjsongraph.js, visualization, new-features
+:category: gsoc
+:lang: en
+:image_url: https://openwisp.org/images/default-preview.webp
```

Comment:
This image doesn't relate to the blog post at all. I sent instructions via email on how to come up with a better one.

The size will have to be specified.